<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wg_demand</class>
 <widget class="QWidget" name="wg_demand">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Button_back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>查询网关号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_equipment">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设备列表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_chatrecord">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>下载聊天记录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_recommended">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>推荐人数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_gatewaydate">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>网关充值</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_adduser">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加第三方用户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_handlelog">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>操作日志</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_gateway">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>辅网关号</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
